packages/framework/src/Controller/Admin/ClosedDayController.php

```diff
@@ -2,26 +2,25 @@
 
 declare(strict_types=1);
 
-namespace App\Controller\Admin;
+namespace Shopsys\FrameworkBundle\Controller\Admin;
 
-use App\Form\Admin\ClosedDayFormType;
-use App\Model\Store\ClosedDay\ClosedDayDataFactory;
-use App\Model\Store\ClosedDay\ClosedDayFacade;
-use App\Model\Store\ClosedDay\Exception\ClosedDayNotFoundException;
-use App\Model\Store\ClosedDay\Grid\ClosedDayGridFactory;
 use Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade;
-use Shopsys\FrameworkBundle\Controller\Admin\AdminBaseController;
+use Shopsys\FrameworkBundle\Form\Admin\Store\ClosedDayFormType;
 use Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider;
+use Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayDataFactory;
+use Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayFacade;
+use Shopsys\FrameworkBundle\Model\Store\ClosedDay\Exception\ClosedDayNotFoundException;
+use Shopsys\FrameworkBundle\Model\Store\ClosedDay\Grid\ClosedDayGridFactory;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class ClosedDayController extends AdminBaseController
 {
     /**
-     * @param \App\Model\Store\ClosedDay\ClosedDayFacade $closedDayFacade
-     * @param \App\Model\Store\ClosedDay\Grid\ClosedDayGridFactory $closedDayGridFactory
-     * @param \App\Model\Store\ClosedDay\ClosedDayDataFactory $closedDayDataFactory
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayFacade $closedDayFacade
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\Grid\ClosedDayGridFactory $closedDayGridFactory
+     * @param \Shopsys\FrameworkBundle\Model\Store\ClosedDay\ClosedDayDataFactory $closedDayDataFactory
```

Comment:
controllers can be overridden, so the properties should be protected

Reply:
fixed :)